src/renderer/pages/Assets/Assets/Assets.tsx

```diff
@@ -30,8 +32,19 @@ export const Assets = () => {
           </div>
         </Header>
         <ShardSelectorButton />
-
-        <AssetsChainView query={query} activeShards={activeShards} hideZeroBalances={hideZeroBalances} />
+        <div className="flex flex-col gap-y-4 w-full h-full overflow-y-scroll">
+          {activeShards.length > 0 && (
```

Comment:
This condition raises same point as previous condition with `AssetView` 😌
We pass `activeShards` to portfolio and can use it to determine if UI should be rendered
For `AssetsChainView` it's passed through props

Reply:
And again I don't like how it looks like, we pass the same prop twice make the view components less readable even though it's a common conditions for the both views 

But if you believe it's will be better overall - we can have it this way